BBC-archive/psammead
# Fix Babel/Storybook Setup

Resolves #115

Storybook should work and all components should load in styles.  Note: Reith font and normalise setup are not yet added. Those will be added in a separate PR.

**Testing instructions**

```
git checkout fix-babel-storybook-setup
npm install 
```
This installs initial dependencies & all individual packages
```
npm run storybook 
```
This runs `clear_styled_components` & then starts storybook

Visit http://localhost:8080  

There should be no errors, and styles should be pulled in.
